CSS/style.css

```diff
@@ -1,3 +1,96 @@
-.new{
-    color:black;
-}
\ No newline at end of file
+* {
+  font-family: "inter", sans-serif;
+  margin: 0;
+  padding: 0;
+  color: #fff;
+}
+
+#paragraph {
+  color: black;
+}
+
+.mobile-view {
+  margin: 8px 18px;
+  height: 48px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: black;
+}
```

Comment:
- [ ] KIndly ensure that your design matches the [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346) chosen 👍. It is good to have a good UI which is a good selling point to your users and potential employers and it's also a good user experience 😄

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/176967414-29e47b96-6c46-4a66-bd44-4417afdceed7.png) | ![image](https://user-images.githubusercontent.com/63473584/176967478-bb837be0-aa60-4a0d-8a05-9ebd819535c3.png)  |

Reply:
@vigehi as previously requested kindly ensure your design matches the Figma design given.